contrib/IBM/sarama/sarama.go

```diff
@@ -0,0 +1,304 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
The path for this needs to be `contrib/IBM/sarama.v1/`

Reply:
I've updated the path.